packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -292,6 +294,7 @@ describe('Overhead controller', () => {
                   if (trace.type === 'web') {
                     urlCounter++
                     expect(trace.meta['_dd.iast.json']).not.to.be.undefined
+                    expect(trace.meta['_dd.iast.enabled']).eq('1')
```

Comment:
I miss a test where `trace.meta['_dd.iast.json']` is undefined and  `trace.meta['_dd.iast.enabled']` is `'1'`

Request without vulnerability, but not ignored by overhead controller.

Reply:
Done: [commit](https://github.com/DataDog/dd-trace-js/pull/2651/commits/3cb0f1bd60fba6d819bb0ec013e663ccd6b2abea)